### PR TITLE
Add Noresqa

### PR DESCRIPTION
**Pull request by shimhz**

### CODE REVIEW

File: versa/scorer_shared.py [hunk 1]
```diff
@@ -495,6 +495,28 @@ def load_score_modules(score_config, use_gt=True, use_gt_text=False, use_gpu=Fal
             }
             logging.info("Initiate srmr successfully")
 
+        elif config["name"] == "noresqa":
```
Comment (ftshijt) [inline versa/scorer_shared.py:498]: You might miss the entry in use_score_modules?

File: versa/utterance_metrics/noresqa.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+def noresqa_model_setup(model_tag="default", metric_type=0, use_gpu=False):
+    if use_gpu:
+        device = "cuda"
+    else:
+        device = "cpu"
+
+    if model_tag == "default":
+
+        if not os.path.isdir("./checkpoints/Noresqa"):
```
Comment (ftshijt) [inline versa/utterance_metrics/noresqa.py:52]: Is it possible to set the cache dir?

I may misunderstood, but if it is wav2vec_small, there are some similar models using it. We may redesign the cache for those in a later timing.

**Review by ftshijt — COMMENTED** [review]
Thanks!

File: versa/utterance_metrics/noresqa.py (hunk 2 above)
Comment (shimhz) [reply versa/utterance_metrics/noresqa.py]: We can set the cache dir by using `urlretrieve`. Yeah we can redesign the cache later~

**Pull request merged**